### PR TITLE
Fix java gradle

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: java-gradle/Makefile [hunk 1]
```diff
@@ -5,15 +5,7 @@ build:
...
-	export ROOKOUT_AGENT_HOST=cloud.agent.rookout.com
-	export ROOKOUT_AGENT_PORT=443
-	export ROOKOUT_TOKEN=$(token)
-	java  -javaagent:rook.jar -jar build/libs/rookoutDemo-1.0.0.jar
-
 gradle-run:
-	export ROOKOUT_AGENT_HOST=cloud.agent.rookout.com
-	export ROOKOUT_AGENT_PORT=443
 	export ROOKOUT_TOKEN=$(token)
 	gradle run
```
Comment (MickaelAlliel) [inline java-gradle/Makefile]: this command will not get ROOKOUT_TOKEN. every command in a Makefile runs in a subshell which do not inherit envs

One solution is to do `export ROOKOUT_TOKEN=$(token) && gradle run`
Other solution is to export token outside of makefile

**Comment by LiranLast** [comment]
@MickaelAlliel @ManuelNemirovsky No more Makefiles, please. Just use Gradle and env variables...

File: java-gradle/Makefile (hunk 1 above)
Comment (ManuelNemirovsky) [reply java-gradle/Makefile]: Done

File: java-gradle/README.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ This sample may be out of date. If you face any issues, please reach out to mail
...
 We have added Rookout to the original project by:
-1. Adding soruces to the project jar(`build.gradle`):
+1. Adding soruces and dependencies to the project jar(`build.gradle`):
     ```properties
     jar {
         from sourceSets.main.allSource
     }
+    
+    dependencies {
+    compile group: 'com.rookout', name: 'rook', version: '0.1.36'
```
Comment (LiranLast) [inline java-gradle/README.md]: @ManuelNemirovsky this should not be a compile dependency.

Maybe `runtime`?

File: java-maven/README.md [hunk 3]
```diff
@@ -10,14 +10,15 @@ This sample may be out of date. If you face any issues, please reach out to mail
...
+    $ git clone https://github.com/Rookout/deployment-examples/tree/master/java-maven
     $ make build
     ```
+2. Export organization token
+     ```bash
+     export ROOKOUT_TOKEN=<Your-Token>
+     ```
 3. Run:
     ```bash
     $ make run
```
Comment (LiranLast) [inline java-maven/README.md]: @ManuelNemirovsky no more makes :(

Comment (ManuelNemirovsky) [reply java-maven/README.md]: it was one of the old java-maven files
my mistake

File: java-gradle/build.gradle [hunk 4]
```diff
@@ -12,6 +12,7 @@ repositories {
 }
 dependencies {
     compile group: 'io.javalin', name: 'javalin', version:'1.3.0'
+    compile group: 'com.rookout', name: 'rook', version: '0.1.36'
```
Comment (LiranLast) [inline java-gradle/build.gradle]: @ManuelNemirovsky runtime?

File: java-maven/README.md [hunk 5]
```diff
@@ -10,17 +10,18 @@ This sample may be out of date. If you face any issues, please reach out to mail
 * [Using docker compose](#using-docker-compose)
 * [Rookout Integration explained](#rookout-integration-explained)
 ## Running locally
-1. Run the Rookout agent:
-    ``` bash
-    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-    ```
-2. Compile the project jar and download the Java agent:
+1. Clone and compile the project jar and download the Java agent:
```
Comment (LiranLast) [inline java-maven/README.md]: @ManuelNemirovsky is this on the same PR intentionally?

Comment (ManuelNemirovsky) [reply java-maven/README.md]: No, 
I got confused between the branches when updating the java-maven readme file, but its supposed to be the same code.

**Comment by LiranLast** [comment]
@ManuelNemirovsky please revert any irrelevant changes PR

File: java-maven/README.md [hunk 6]
```diff
@@ -52,6 +52,6 @@ We have added Rookout to the original project by:
         java  -javaagent:rook.jar -jar target/rookoutDemo-1.0.0.jar 
     ```
 
-[Java + Rookout]: https://docs.rookout.com/docs/rooks-setup.html
+[Java + Rookout]: https://docs.rookout.com/docs/installation-java.html
```
Comment (LiranLast) [inline java-maven/README.md]: @ManuelNemirovsky this file doesn't belong here :)

**Comment by ManuelNemirovsky** [comment]
@LiranLast Fixed :)

File: java-maven/README.md (hunk 5 above)
Comment (LiranLast) [reply java-maven/README.md]: @ManuelNemirovsky and this file?

File: java-gradle/README.md [hunk 7]
```diff
@@ -31,17 +31,24 @@ This sample may be out of date. If you face any issues, please reach out to mail
 This example is based of the Java javalin "Hello-World" example available [here].
 
 We have added Rookout to the original project by:
-1. Adding soruces to the project jar(`build.gradle`):
+1. Adding soruces and dependencies to the project jar (`build.gradle`):
```
Comment (MickaelAlliel) [inline java-gradle/README.md]: typo - sources

Comment (ManuelNemirovsky) [reply java-gradle/README.md]: Done

File: java-gradle/README.md [hunk 8]
```diff
@@ -10,17 +10,17 @@ This sample may be out of date. If you face any issues, please reach out to mail
...
+    $ git clone https://github.com/Rookout/deployment-examples/tree/master/java-gradle
     ```
+2. Export organization token:
+     ```bash
+     export ROOKOUT_TOKEN=Your-Token
+     ```
 3. Run using gradle:
     ```bash
-    $ make gradle-run token=YOUR_TOKEN
+    $ gradle run
```
Comment (MickaelAlliel) [inline java-gradle/README.md]: Use command `./gradlew run`
This uses the gradle wrapper (basically gradle bundled in the repo) so if someone does not have gradle installed they can still run the application

Comment (ManuelNemirovsky) [reply java-gradle/README.md]: Done

**Pull request merged**